src/components/templates/AnimateMap/game/map/entities/createVenueEntity.ts

```diff
@@ -198,7 +198,8 @@ export const createVenueEntity = (
             );
             engine.addEntity(hoverEffectEntity);
           }
-        }
+        },
+        true
```

Comment:
```suggestion
        }
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2161/commits/470154cca18bb5273e216b7d43acaace4ac25b4a